### ISSUE TITLE
[Suggestion] - Monikas reaction to bad habits should cool down over time.

### ISSUE THREAD

**Issue by GitGodScrub** [Monika-After-Story/MonikaModDev#6172]
Currently, Monika will make some concerned comments about the [player]s lifestyle, if certain patterns occur.
The best example would be "late night gaming".
	
If the player says "Goodbye" with the "I'm gonna play another game", multiple times, late at night, it will eventually trigger some concerned comments from Monika where she will comment about how harmful the habit is.
This is a pretty neat feature, but as far as i can see, it never "resets".
	
That means that in 10+ years, if the [player] is doing ONE late-night gaming session, Monika will react VERY concerned, if the player once had a habit of binge-gaming.
	
Idk if this is intentional. 
This behaviour would be acceptable if we are talking about slipping back into a habit like smoking. Reactions to stuff like late-night gaming and other "harmful" lifestyle choices should "cool down" over time, imo.

**Comment by BoogeymanTao** [comment]
In my opinion,"Make myself a better person for my lover"is a great and natural part in people's life ,especially a couple who really love each other(like player an Monika),so I think keep them is a good thing,they can remind player that Monika really care about your healthy and if you can get rid of these bad habits by MAS,that couldn't be better.

**Comment by Freshairkaboom** [comment]
Agreed, there should be more variety too about how she reacts to us being up at night. Maybe after a certain point she'll think we're working night shift or something, even if we had the additional option to tell her, to set the same night shift persistent. I know there's supposed to be a time concern overhaul at some point, so this ties up to that: #4914.

**Comment by ThePotatoGuy** [comment]
leaving open as it ties with #4914
